### PR TITLE
refac export.create section

### PR DESCRIPTION
**Pull request by EdouardParis**

### CODE REVIEW

File: content/reference/export.md [hunk 1]
```diff
@@ -50,10 +53,75 @@ Creates an export for the project with the given ID. This endpoint is only acces
...
-| `type`   | string | Type of the export -- required, must be `orders`          |
+| Field     | Type              | Description                                               |
+| --------- | ----------------- | --------------------------------------------------------- |
+| `format`  | string            | Format of the export -- required, must be `csv` or `xlsx` |
+| `type`    | string            | Type of the export -- required, must be `orders`          |
+| `columns` | columns payload   | Columns to be exported                                    |
+
+**columns payload for order export**
+
+| Field                              | Type    | Description                                                    |
```
Comment (yansal) [inline content/reference/export.md]: Either fill the description column for each row, or just remove it

File: content/reference/export.md [hunk 2]
```diff
@@ -50,10 +53,75 @@ Creates an export for the project with the given ID. This endpoint is only acces
...
-| -------- | ------ | --------------------------------------------------------- |
-| `format` | string | Format of the export -- required, must be `csv` or `xlsx` |
-| `type`   | string | Type of the export -- required, must be `orders`          |
+| Field     | Type              | Description                                               |
+| --------- | ----------------- | --------------------------------------------------------- |
+| `format`  | string            | Format of the export -- required, must be `csv` or `xlsx` |
+| `type`    | string            | Type of the export -- required, must be `orders`          |
+| `columns` | columns payload   | Columns to be exported                                    |
+
+**columns payload for order export**
```
Comment (yansal) [inline content/reference/export.md]: Make it consistent with the other headers, and call it something like `Export columns`

File: content/reference/export.md [hunk 3]
```diff
@@ -50,10 +53,75 @@ Creates an export for the project with the given ID. This endpoint is only acces
...
 
-| Field    | Type   | Description                                               |
-| -------- | ------ | --------------------------------------------------------- |
-| `format` | string | Format of the export -- required, must be `csv` or `xlsx` |
-| `type`   | string | Type of the export -- required, must be `orders`          |
+| Field     | Type              | Description                                               |
+| --------- | ----------------- | --------------------------------------------------------- |
+| `format`  | string            | Format of the export -- required, must be `csv` or `xlsx` |
+| `type`    | string            | Type of the export -- required, must be `orders`          |
+| `columns` | columns payload   | Columns to be exported                                    |
```
Comment (yansal) [inline content/reference/export.md]: columns payload should be called something like `Export columns` and link to the appropriate header

File: content/reference/export.md [hunk 4]
```diff
@@ -41,6 +41,9 @@ Retrieves the export with the given ID. This endpoint is only accessible to the
 ## Create an export
 
 Creates an export for the project with the given ID. This endpoint is only accessible to the project owner.
+With the field `columns` in the payload, the headers of the export can be filtered and specified with an other title.
```
Comment (yansal) [inline content/reference/export.md]: With the `columns` payload field, the export columns can be filtered and renamed.

File: content/reference/export.md [hunk 5]
```diff
@@ -41,6 +41,9 @@ Retrieves the export with the given ID. This endpoint is only accessible to the
...
+Each field included in the export must be specified with its column name and the wanted header title.
+If any columns is specified in the payload, all the columns with their default header title will be included in the export.
```
Comment (yansal) [inline content/reference/export.md]: Does it mean that all the columns are always included? If yes it contradicts the first sentence.

Comment (EdouardParis) [reply content/reference/export.md]: sorry mistakes from me `if no columns`

File: content/reference/export.md (hunk 1 above)
Comment (EdouardParis) [reply content/reference/export.md]: yes, work in progress, I wanted to know if want fields to be described or not ?

Comment (yansal) [reply content/reference/export.md]: I think it's ok to _not_ add descriptions here because most fields are already described in other resources

**Pull request merged**